mmocr/models/textdet/postprocess/abcnet_postprocessor.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+# from mmcv.runner import force_fp32
+from mmcv.ops import batched_nms
+from mmdet.core import build_bbox_coder, multi_apply
+from mmdet.core.anchor.point_generator import MlvlPointGenerator
+from mmdet.core.utils import filter_scores_and_topk, select_single_mlvl
+
+from mmocr.models.builder import POSTPROCESSOR
+from mmocr.utils.box_util import bezier_to_polygon
+from .base_postprocessor import BaseTextDetPostProcessor
+
+
+@POSTPROCESSOR.register_module()
+class ABCNetTextDetProcessor(BaseTextDetPostProcessor):
+
+    def __init__(self,
+                 num_classes=1,
+                 use_sigmoid_cls=True,
+                 strides=(4, 8, 16, 32, 64),
+                 bbox_coder=dict(type='DistancePointBBoxCoder'),
+                 text_repr_type='poly',
+                 train_cfg=None,
+                 test_cfg=None,
+                 **kwargs):
+        super().__init__(
+            text_repr_type=text_repr_type,
+            train_cfg=train_cfg,
+            test_cfg=test_cfg,
+            **kwargs)
+        self.prior_generator = MlvlPointGenerator(strides)
+        self.bbox_coder = build_bbox_coder(bbox_coder)
+        self.use_sigmoid_cls = use_sigmoid_cls
+        if self.use_sigmoid_cls:
+            self.cls_out_channels = num_classes
+        else:
+            self.cls_out_channels = num_classes + 1
+
+    # @force_fp32(apply_to=('cls_scores', 'bbox_preds'))
+
+    def filter_and_location(self,
+                            det_results,
+                            img_meta=None,
+                            nms_pre=-1,
+                            score_thr=0,
+                            max_per_img=100,
+                            nms=dict(type='IoULoss', loss_weight=1.0),
```

Comment:
```suggestion
                            nms_cfg=dict(iou_thr=0.5),
```
We may need `train_cfg` and `test_cfg` to specify different threshold values